src/NewCommand.php

```diff
@@ -40,10 +41,36 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+
```

Comment:
Unnecessary gap

Reply:
I haven't coding standards'ed (it's a word!) anything yet, i will once the feedback is in on filenames / handling etc.